migrations/2_deploy_organization.js

```diff
@@ -30,50 +31,48 @@ module.exports = async (deployer) => {
   const orgName = "Genesis";
   const tokenName = "Gen";
   const tokenSymbol = "GEN";
-  const founders = [web3.eth.accounts[0]];
-  const initRepInWei = [web3.toWei(10)];
-  const initTokenInWei = [web3.toWei(1000)];
   const orgNativeTokenFee = 0;
   const defaultVotingMachineParams = {
     preBoostedVoteRequiredPercentage: 50,
-    preBoostedVotePeriodLimit: 60,
-    boostedVotePeriodLimit: 60,
-    thresholdConstA: 1,
-    thresholdConstB: 1,
+    preBoostedVotePeriodLimit: 5184000, // 2 months
```

Comment:
from where these params come from ?

Reply:
They come from Adam and Tiber, as far as I know.  They are not complete.

Reply:
@leviadam ?

Reply:
yup, Adam gave them to me, still gotta figure out the threshold params